src/modules/users/user.controller.ts

```diff
@@ -1,37 +1,56 @@
-import { Controller, Delete, Get, Post, Put } from '@nestjs/common';
+import { Body, Controller, Delete, Get, Param, Post, Put, ValidationPipe } from '@nestjs/common';
 import { UserService } from './user.service';
-// Nhan cac req tu clien -> server va response server -> clienr (data from service)
+import { ResponseData } from 'modules/global/globalClass';
+import { HttpStatus, HttpMessage } from 'modules/global/globalEnum';
+import { User } from 'models/user.model';
+import { UserDTO } from 'dto/user.dto';
+
 @Controller('users')
 export class UserController {
   constructor(private readonly userService: UserService) {}
 
   @Get()
-  getUsers(): string {
-    return this.userService.getUsers();
-  }
-
-  @Get('/check-health')
-  checkHealth(): string {
-    return 'Ok';
+  getUsers(): ResponseData<User[]> {
+    try {
+      return new ResponseData<User[]>(this.userService.getUsers(), HttpStatus.SUCCESS, HttpMessage.SUCCESS);
+    } catch (error) {
+      return new ResponseData<User[]>(null, HttpStatus.ERROR, HttpMessage.ERROR);
```

Comment:
nghiên cứu lại chỗ này nhé vì midnh sẽ có 
404 not found
400 validate error 
và 502 là lỗi server
trả về lỗi 404 cho mọi case là chưa đúng lắm
e xem lại nhé